### ISSUE TITLE
Getting broken image for predicted value at exp.show_in_notebook stage

### ISSUE THREAD

**Issue by hojun0705** [marcotcr/lime#301]
Hello! 

I am using tensorflow keras API for deep learning. the model has been built with Keras.
I do not know why my rendering section shows me not correctly on 'predicted value'. I thought I did a fine job. Can anyone think of why? Thank you.
![image](https://user-images.githubusercontent.com/38139484/53924785-bec52380-40c0-11e9-926e-f27e8bfd24f1.png)

**Comment by jkrabbenbos** [comment]
Happened to me too when the predicted value was negative. When I translated it to a positive number, it showed correctly.

**Comment by hojun0705** [comment]
> Happened to me too when the predicted value was negative. When I translated it to a positive number, it showed correctly.

Thank you for your comment. But as you see, my prediction is not negative. I do not know what causes this problem to me yet.

**Comment by hiyam97** [comment]
I have the same problem as well. Did anyone know the answer yet ?

**Comment by JiaHong-Lee** [comment]
Hi, I have this issue as well. This can be reproduced using the following code:
```python3
from sklearn.linear_model import LinearRegression
from sklearn.datasets import load_boston
from sklearn.model_selection import train_test_split
from lime.lime_tabular import LimeTabularExplainer

boston = load_boston()
X = boston.data
y = boston.target
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.4, random_state=0)

linr = LinearRegression()
linr.fit(X_train, y_train)

explainer = LimeTabularExplainer(X_train, feature_names=list(boston.feature_names),
                                class_names=['MEDV'],
                                mode='regression',
                                random_state=0)
exp = explainer.explain_instance(X_test[25], linr.predict, num_features=5)
print(f"Predicted min: {exp.min_value}")
print(f"Predicted actual: {exp.predicted_value}")
print(f"Predicted max: {exp.max_value}")
exp.show_in_notebook(show_table=True)
```

Result:
![IMG_0396](https://user-images.githubusercontent.com/24759194/76276725-9ec2d400-62c1-11ea-9bba-50f4da8fe34c.png)


After digging into the code, the problem lies in the `bundle.js` code line 27391 where `Math.log10(min_value)` evaluates to NaN when `min_value` is negative, causing the variable `num_digits` and `corner_width` to be NaN and `bar_width` to be negative. Maybe a fix is to take `Math.abs` as mentioned by @jkrabbenbos. I still don’t understand why the min predicted value is negative though.

**Comment by marcotcr** [comment]
Fixed in 5c7107e09e85c07a3417686673f2c81756c29280.
Thanks @JiaHong-Lee for identifying the cause of the bug and providing a reproducible example.
